libtransmission/peer-msgs.cc

```diff
@@ -2403,15 +2393,14 @@ void tr_peerMsgsImpl::sendPex()
     }
 
     /* write the pex message */
-    auto* const payload = tr_variantToBuf(&val, TR_VARIANT_FMT_BENC);
-    evbuffer_add_uint32(out, 2 * sizeof(uint8_t) + evbuffer_get_length(payload));
```

Comment:
May be justified, but what was removed for the `2 *` to be removed?

Reply:
It was removed in error; thanks for spotting this :100: 

Fixed in e7a39d8b795865c9d89f3e9db47eb70f510d6d93